pkg/operator/k8sutil/kvstore.go

```diff
@@ -40,16 +41,16 @@ func NewConfigMapKVStore(namespace string, clientset kubernetes.Interface) *Conf
 func (kv *ConfigMapKVStore) GetValue(storeName, key string) (string, error) {
 	cm, err := kv.clientset.CoreV1().ConfigMaps(kv.namespace).Get(storeName, metav1.GetOptions{})
 	if err != nil {
-		if errors.IsNotFound(err) {
-			return "", kvstore.NewNotExistError(storeName, key)
-		}
-
 		return "", err
 	}
 
 	val, ok := cm.Data[key]
 	if !ok {
-		return "", kvstore.NewNotExistError(storeName, key)
+		return "", &errors.StatusError{ErrStatus: metav1.Status{
```

Comment:
is this the idiomatic way to create a k8s error that would return true for `errors.IsNotFound(err)`?

Reply:
i was meaning to look if there was a better way to do this